c++/src/Compression.cc

```diff
@@ -207,7 +214,7 @@ namespace orc {
     // PASS
   }
```

Comment:
```cpp
 CompressionStream::CompressionStream(OutputStream* outStream, int compressionLevel,
                                       uint64_t compressionBlockSize, uint64_t memoryBlockSize, MemoryPool& pool,
                                       WriterMetrics* metrics) 
```
capacity -> compressionBlockSize, blockSize->memoryBlockSize
Same for other compression streams.